src/highdicom/pr/content.py

```diff
@@ -0,0 +1,634 @@
+        if not isinstance(order, int):
+            raise TypeError('"order" must be an integer.')
+        self.GraphicLayerOrder = order
+        if description is not None:
+            _check_long_string(description)
+            self.GraphicLayerDescription = description
+        if display_color is not None:
+            if not isinstance(display_color, CIELabColor):
+                raise TypeError(
+                    '"recommended_display_color" must be of type '
+                    'highdicom.color.CIELabColor.'
+                )
+            self.GraphicLayerRecommendedDisplayCIELabValue = list(
+                display_color.value
+            )
+
+
+class GraphicGroup(Dataset):
+
+    """Dataset describing a grouping of annotations.
+
+    Note
+    ----
+    Graphic Groups are an independent concept from Graphic Layers. Where a
+    Graphic Layer (:class:`highdicom.pr.GraphicLayer`) specifies which
+    annotations are rendered first, a Graphic Group specifies which annotations
+    belong together and shall be handled together (e.g., rotate, move)
+    independent of the Graphic Layer to which they are assigned.
+
+    Each annotation (:class:`highdicom.pr.GraphicObject` or
+    :class:`highdicom.pr.TextObject`) may optionally be assigned to a
+    GraphicGroup upon construction (whereas assignment to a
+    :class:`highdicom.pr.GraphicLayer` is required.)
+
+    """
+    def __init__(
+        self,
+        graphic_group_id: int,
+        label: str,
+        description: Optional[str] = None
+    ):
+        """
+
+        Parameters
+        ----------
+        graphic_group_id: int
+            A positive integer that uniquely identifies this graphic group.
+        label: str
+            Name used to identify the Graphic Group (maximum 64 characters).
+        description: Union[str, None], optional
+            Description of the group (maxiumum 10240 characters).
+
+        """
+        super().__init__()
+        if not isinstance(graphic_group_id, int):
+            raise TypeError(
+                'Argument "graphic_group_id" must be an integer.'
+            )
+        if graphic_group_id < 1:
+            raise ValueError(
+                'Argument "graphic_group_id" must be a positive integer.'
+            )
+        self.GraphicGroupID = graphic_group_id
+        _check_long_string(label)
+        self.GraphicGroupLabel = label
+        if description is not None:
+            _check_short_text(description)
+            self.GraphicGroupDescription = description
+
+    @property
+    def graphic_group_id(self) -> int:
+        """int: The ID of the graphic group."""
+        return self.GraphicGroupID
+
+
+class GraphicObject(Dataset):
+
+    """Dataset describing a graphic annotation object."""
+
+    def __init__(
+        self,
+        graphic_type: Union[GraphicTypeValues, str],
+        graphic_data: np.ndarray,
+        units: Union[AnnotationUnitsValues, str] = AnnotationUnitsValues.PIXEL,
```

Comment:
Do we really want to provide a default here? I think it would be better to let the caller explicitly specify the unit so that they are aware of the fact that there are different units. This is such a complex subject that I am concerned a default will do more harm than good. What do you think?

Reply:
Removed default values here (and for `TextObject` constructor) in 2b40cc1